.env

```diff
@@ -1,12 +1,14 @@
 CHROME_HOSTNAME=chrome
+COMPOSE_DOCKER_CLI_BUILD=1
```

Comment:
@aprilrieger this is the secret to eternal youth. or maybe it was faster builds. it was one of those two things